lib/deploy/storage/deploy.js

```diff
@@ -1,25 +1,25 @@
 'use strict';
 
 var _ = require('lodash');
+var chalk = require('chalk');
 var RSVP = require('rsvp');
 
 var gcp = require('../../gcp');
 var utils = require('../../utils');
 
 module.exports = function(context, options) {
-  var files = _.get(context, 'storage.rules');
-  if (!files) {
+  var rulesMap = _.get(context, 'storage.rulesMap');
+  if (!rulesMap) {
     return RSVP.resolve();
   }
 
-  return gcp.storage.buckets.getDefault(options.project).then(function(defaultBucket) {
-    if (!defaultBucket) {
-      return utils.reject('Unable to locate your default bucket. Please visit the "Storage" panel in the Firebase console and check for errors.');
-    }
-
-    context.storage.defaultBucket = defaultBucket;
-    return gcp.rules.createRuleset(options.project, files);
-  }).then(function(rulesetName) {
-    context.storage.rulesetName = rulesetName;
+  var promises = [];
+  _.forEach(rulesMap, function(rules, filename) {
+    utils.logBullet(chalk.cyan('storage:') + ' uploading rules ' + chalk.bold(filename) + '...');
+    promises.push(gcp.rules.createRuleset(options.project, rules.files).then(function(rulesetName) {
+      rules.ruleset = rulesetName;
```

Comment:
What is the purpose of this line?

Reply:
It persists the created ruleset name in the `context` so the release phase can access it when performing a release. See https://github.com/firebase/firebase-tools/pull/476/files#diff-636795e647db388748a86f13457eb5ebR22